js/src/data/selectors.js

```diff
@@ -498,3 +498,25 @@ export const getAdsRecommendations = ( state, types, campaign_id = null ) => {
 	const key = arrayToUnderscoreKey( keyToHash );
 	return state.ads.recommendations[ key ] || null;
 };
+
+/**
+ * Get detailed error objects whose `slot` matches any of the provided slots.
+ *
+ * If `errorSlots` is not an array or is empty, the function returns null.
```

Comment:
```suggestion
 * If `slots` is not an array or is empty, the function returns null.
```
Please review comments to use the updated param names @ankitrox